functions/createField.js

```diff
@@ -32,6 +33,7 @@ export function createField() {
             coordinateY--;
         }
         cell.id = formatId(coordinateX, coordinateY);
+        params.collectionId = params.collectionId.add(cell.id);

```

Comment:
зачем присвоение ?
Просто вот так использовать
```suggestion
  params.collectionId.add(cell.id);
```